src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -981,7 +997,8 @@ public int getIntValue() {
                 JDBCType.Category.CHARACTER,
                 JDBCType.Category.LONG_CHARACTER,
                 JDBCType.Category.NCHARACTER,
-                JDBCType.Category.LONG_NCHARACTER)),
+                JDBCType.Category.LONG_NCHARACTER,
+                JDBCType.Category.SQL_VARIANT)), 
```

Comment:
I am just wondering why we only add setter conversion support for SQL_Variant to Numeric? not other types?